transmart-core-api/src/main/groovy/org/transmartproject/core/ontology/OntologyTerm.groovy

```diff
@@ -120,6 +120,13 @@ public interface OntologyTerm {
      */
     List<Patient> getPatients()
 
+    /**
+     * Fetches number of patients with at least one observation for this OntologyTerm.
+     *
+     * @return number of patients
+     */
+    int getPatientCount()
```

Comment:
Should be part of a service instead of (the interface of) domain classes. I added a comment to `AbstractI2b2Metadata.groovy` on the same issue.